### PR TITLE
frontend-feat: toll and feedback UI and flow changes

### PR DESCRIPTION
**Pull request by spravinkumar9952**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

- Carousel Issue cards
- Toll popups
- other ui fixes

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added integration tests for my changes where possible
- [ ] I tested the changes using integration tests
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-driver/src/Screens/HomeScreen/ScreenData.purs [hunk 1]
```diff
@@ -261,6 +261,8 @@ initData =
       , odometerImageUploading: false
       , showAcWorkingPopup: Nothing
       , acExplanationPopup : false
+      , showTollChargePopup : true
+      , showTollChargeAmbigousPopup : false
```
Comment (ashishsingh101) [inline Frontend/ui-driver/src/Screens/HomeScreen/ScreenData.purs]: we can create new object for toll and move both of them inside it

File: Frontend/ui-driver/src/Screens/HomeScreen/View.purs [hunk 2]
```diff
@@ -550,6 +553,7 @@ getCarouselConfig view state = {
   , onPageScrollStateChanged : Just BannerStateChanged
   , onPageScrolled : Nothing
   , currentIndex : state.data.bannerData.currentBanner
+  , showScrollIndicator : true
```
Comment (ashishsingh101) [inline Frontend/ui-driver/src/Screens/HomeScreen/View.purs:556]: it should be based on length of banners

**Review by ashishsingh101 — CHANGES_REQUESTED** [review]

File: Frontend/ui-driver/src/Screens/HomeScreen/PopUpConfig.purs
[the remark this answers is not in the document]
Comment (spravinkumar9952) [reply Frontend/ui-driver/src/Screens/HomeScreen/PopUpConfig.purs:1]: Component Config is already growing too much. compilation time is increased. thats why. (like banner config)

File: Frontend/ui-customer/src/Components/DriverInfoCard/View.purs [hunk 3]
```diff
@@ -767,6 +767,7 @@ getCarouselConfig view state = {
   , onPageScrollStateChanged : Just BannerStateChanged
   , onPageScrolled : Nothing
   , currentIndex : state.data.bannerData.currentBanner
+  , showScrollIndicator : true
```
[the remark this answers is not in the document]
Comment (spravinkumar9952) [reply Frontend/ui-customer/src/Components/DriverInfoCard/View.purs:770]: be default we need this scroll indicator in all the places. thats why made it as true. in my case only its false.

File: Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/PopUpConfigs.purs
[the remark this answers is not in the document]
Comment (spravinkumar9952) [reply Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/PopUpConfigs.purs:1]: Component Config is already growing too much. compilation time is increased. thats why. (like banner config)

File: Frontend/ui-driver/src/Screens/HomeScreen/View.purs (hunk 2 above)
Comment (spravinkumar9952) [reply Frontend/ui-driver/src/Screens/HomeScreen/View.purs:556]: thats handled internally

**Review by ashishsingh101 — DISMISSED** [review]

**Review by aryan-13 — DISMISSED** [review]

**Review by Vignesh-772 — DISMISSED** [review]

**Review by Vignesh-772 — APPROVED** [review]

**Pull request merged**